MechJeb2/OrbitExtensions.cs

```diff
@@ -285,15 +285,15 @@ public static double AscendingNodeTrueAnomaly(this Orbit a, Orbit b)
 
         //Gives the true anomaly (in a's orbit) at which a crosses its descending node
         //with b's orbit.
-        //The returned value is always between 0 and 360.
+        //The returned value is always between 0 and 2 * PI.
         public static double DescendingNodeTrueAnomaly(this Orbit a, Orbit b)
         {
-            return MuUtils.ClampDegrees360(a.AscendingNodeTrueAnomaly(b) + 180);
+            return MuUtils.ClampRadiansTwoPi(a.AscendingNodeTrueAnomaly(b) + Math.PI);
```

Comment:
This is probably the wrong way to go about doing this.  Most of MJ internally uses degrees instead of Radians, and without going through and auditing all the consumers of these APIs the right way to fix it is probably to return degrees everywhere.  Generally most of the Unity and KSP APIs also return degrees instead of radians.

Radians are objectively mathematically better, but its easier not to fight it.

Reply:
Also are you sure that this returns radians?  It looks like it calls TrueAnomalyFromVector() which looks like it uses degrees.

Reply:
There are two `TrueAnomalyFromVector` - one in `OrbitExtension` which takes two arguments, and another one in the KSP DLL which takes only one argument. The KSP one returns radians. This is the KSP one.

Now, this is a huge problem. From what I understand - this is the first time I am touching anything inside a KSP mod - KSP used to have degrees everywhere but later switched to radians for the true and mean anomaly - variables that are never visible to the user in the stock game.

There is a TODO that I linked in the issue about switching true/mean anomaly to radians to align to the game.

Anyway, currently those functions work in radians, even if the comment says degrees. And the DN was way off, now it is ok. The DN was not displayed before my other PR - which explains why it went unnoticed.

In order to minimize the impact, I decided to keep those functions in radians and modify only the comment and the DN which was simply wrong.

Reply:
Ah, the same-named functions tripped me up.